### PR TITLE
[ExtensionBundle] Updating Extension Bundle to 1.6.0

### PR DESCRIPTION
**Pull request by davidmrdavid**
Releasing new Host versions.
[V1 Release](https://azfunc.visualstudio.com/Azure%20Functions/_releaseProgress?_a=release-pipeline-progress&releaseId=2603)

---

This PR upgrades the V1 extension bundles from version 1.5.0 to 1.6.0. It was done via search-and-replace for all instances of `EXTENSION_BUNDLE_VERSION=1.5.0` to `EXTENSION_BUNDLE_VERSION=1.6.0`.

### PR information

- [x] The title of the PR is clear and informative.
- [x] There are a small number of commits, each of which has an informative message. This means that previously merged commits do not appear in the history of the PR. For information on cleaning up the commits in your pull request, [see this page](https://github.com/Azure/azure-powershell/blob/master/documentation/development-docs/cleaning-up-commits.md).
- [x] If applicable, the PR references the bug/issue that it fixes in the description.
- [ ] New Unit tests were added for the changes made and CI is passing.

### CODE REVIEW

File: host/2.0/stretch/amd64/dotnet/dotnet-slim.Dockerfile [hunk 1]
```diff
@@ -13,7 +13,7 @@ RUN BUILD_NUMBER=$(echo ${HOST_VERSION} | cut -d'.' -f 3) && \
     mv /azure-functions-host/workers /workers && mkdir /azure-functions-host/workers && \
     rm -rf /root/.local /root/.nuget /src
 
-RUN EXTENSION_BUNDLE_VERSION=1.5.0 && \
+RUN EXTENSION_BUNDLE_VERSION=1.6.0 && \
     EXTENSION_BUNDLE_FILENAME=Microsoft.Azure.Functions.ExtensionBundle.1.5.0_linux-x64.zip && \
```
Comment (TsuyoshiUshio) [inline host/2.0/stretch/amd64/dotnet/dotnet-slim.Dockerfile:17]: It looks wrong. Make it 1.6.0 

This is the error message of CI

```
https://functionscdn.azureedge.net/public/ExtensionBundles/Microsoft.Azure.Functions.ExtensionBundle/1.6.0/Microsoft.Azure.Functions.ExtensionBundle.1.5.0_linux-x64.zip 
```

Comment (davidmrdavid) [reply host/2.0/stretch/amd64/dotnet/dotnet-slim.Dockerfile]: Thank you @TsuyoshiUshio , this was indeed the fix :)

**Comment by vrdmr** [comment]
qq @davidmrdavid - Did you use a script to update the extension version?

**Comment by davidmrdavid** [comment]
@vrdmr: I just did a search-and-replace operation, I'm not aware of a better process here. If there is, I'm happy to go for that instead

**Comment by vrdmr** [comment]
@davidmrdavid - We have couple of scripts to update host version - https://github.com/Azure/azure-functions-docker/blob/master/host/update-version.sh

Can you please create a new script - `update-extension-bundles-version.[sh/ps1]` - Not a blocker for this PR but would help is in the long run.

**Review by vrdmr — APPROVED** [review]

**Comment by davidmrdavid** [comment]
Thanks, happy to contribute a script as well. Working on it now

**Pull request merged**